test/functional/rpc_createmultisig.py

```diff
@@ -194,13 +194,19 @@ def do_multisig(self, nkeys, nsigs, output_type, wallet_multi):
         assert_raises_rpc_error(-8, "redeemScript/witnessScript does not match scriptPubKey", node2.signrawtransactionwithkey, rawtx, priv_keys[0:nsigs-1], [prevtx_err])
 
```

Comment:
Lacking before this change as well, can we add a comment here just like there is one in all the code blocks above and below?

`sign the rawtx with nsigs-1 signatures, then sign with the last sig, and then combine to create fully signed tx`